site/src/atomics/display.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Display Atomics
+description: Display related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Display Atomics
+
+The following displays are covered: `block`, `flex`, `inline`, `inline-block`, `inline-flex`, `grid`, and `none`.
+
+The following breakpoints are supported for each display value:
+
+```Text
+.display-{value}-tablet
+.display-{value}-desktop
+.display-{value}-widescreen
+```
+
+## Example
```

Comment:
Can we make this more uniform with the other article? Is this `Usage`?

Reply:
Updated